docs/features/asc1/stateless/walkthrough.md

```diff
@@ -231,18 +231,25 @@ b end_program
 
 ## Opt-In ASC1 depolyment
 
-We get the Contracr Address compiling its TEAL logic:
+We get the Contract Address by compiling its TEAL logic:
 
 ```
 goal clerk compile contract.teal
 
 contract.teal: CONTRACT_ADDRESS
 ```
 
-As any other Algorand Account, the Contract Account must be initialized:
+As any other Algorand Account, the Contract Account must be first initialized with 0,1 ALGO and funded with additional 0,1 ALGO for each additional ASA to Opt-In.
```

Comment:
Please use 0.1 ALGO (rather than 0,1 for precision numbers)

Reply:
Oh! That's my European habit! Done!